### ISSUE TITLE
Screen problem with resolutions under 1080p in desktop

### ISSUE THREAD

**Issue by walkerdeath** [XITRIX/Moonlight-Switch#33]
Hello! First of all, awesome work with this fork, works like a charm and the ui it's awesome!
I've been playing for a couple of months streaming games to my switch at 720p with no problem, but the underlying desktop resolution was 1080p. Today I've tried to change the desktop resolution to 720p as it had no point to be greater, and encountered this strange issue, as you can see in the picture of the next link:
[https://imgur.com/a/bjhL24e](url)
I would say it's like the app thinks the switch display is 1080p and when it tries to output 720p fails to do it at fullscreen. Also the mouse won't go further than the cropped desktop, as if that was the full screen size.
If I can provide anything to help you further ask for it! Thank you!

**Comment by XITRIX** [comment]
Wow, it looks weird, I've never met a problem like that. My desktop automatically change it's screen resolution when I'm start streaming on 720p, it's Nvidia's Experience work, and it should do that. The only solution I can see right now, is to try to reinstall Experience, cause it looks like it behaves incorrect. Also it should scale the picture to the fullscreen no matter what resolution is chosen. Also, does it looks the same on your primary monitor?

**Comment by walkerdeath** [comment]
Okay, I found the solution in the moonlight-stream git, it is very weird, changing the resolution via windows doesn't work, but changing it through nvidia's control panel works just fine!
The link to the github issue where I found the solution: https://github.com/moonlight-stream/moonlight-android/issues/588
I'm sorry, I should have search for it before asking here! Thank you very much and again, great job with this moonlight port!
